### ISSUE TITLE
The template is incompatible with Zabbix 3.4.7

### ISSUE THREAD

**Issue by tiborkiss** [MogiePete/zabbix-systemd-service-monitoring#9]
After commenting out many tags, which were unexpected, now I see missing tags also.
It is possible to have a backported version from this template?

**Comment by MogiePete** [comment]
Can you provide details? This template was originally created in 3.4.x and no modification was done to support 4.x

Additionally, as is listed in the README, this template is currently intended for Zabbix 4.0. As I am no longer running a Zabbix 3.x installation I can't provide assistance in troubleshooting any issues you have without your assistance.

**Comment by tiborkiss** [comment]
The current error says the following:

Invalid tag "/zabbix_export/templates/template(1)/discovery_rules/discovery_rule(1)/item_prototypes/item_prototype(1)": the tag "master_item_prototype" is missing.

Before that I had to remove some tags (many), but now I have to guess the missings.

Unfortunately I am not familiar with Zabbix's template details. Surprisingly I couldn't find a converter between versions.

**Comment by tiborkiss** [comment]
Finally I solved the problem. Temporarily I created an AWS instance with Zabbix 4.0, imported there the template, then manually I recreated the template in my zabbix 3.4 to look like in zabbix 4.0.
I also exported, zipped and attached here for reference.

Then I did some tests and I am happy with this zabbix agent extension.
 
[Template_App_systemd_Services-v3.4.xml.zip](https://github.com/MogiePete/zabbix-systemd-service-monitoring/files/3054610/Template_App_systemd_Services-v3.4.xml.zip)
